deployments/helm/nvidia-device-plugin/templates/daemonset-device-plugin.yml

```diff
@@ -184,6 +194,9 @@ spec:
             value: compute,utility
         securityContext:
           {{- include "nvidia-device-plugin.securityContext" . | nindent 10 }}
+          capabilities:
+            add:
+              - SYS_ADMIN
```

Comment:
Why was this added? There was an issue #856 which should be resolved in the latest release.